documentation/guide-access-control.asciidoc

```diff
@@ -45,6 +45,18 @@ For further details see https://docs.spring.io/spring-security/site/docs/current
 Further, you need to provide an implementation of the https://docs.spring.io/spring-security/site/docs/4.2.x/apidocs/org/springframework/security/core/userdetails/UserDetailsService.html[UserDetailsService] interface.
 A good starting point comes with our application template.
 
+For link:quarkus.asciidoc[quarkus], you can use different extensions for support of different authentication mechanisms. For example:
+
+    * https://quarkus.io/guides/security-properties[`quarkus-elytron-security-properties-file`] provides support for simple properties files that can be used for testing security. This supports both embedding user information in `application.properties` and standalone properties files.
+    * https://quarkus.io/guides/security-jpa[`quarkus-security-jpa`] provides support for authenticating via JPA.
+    * https://quarkus.io/guides/security-jdbc[`quarkus-elytron-security-jdbc`] provides support for authenticating with JDBC
+    * https://quarkus.io/guides/security-jwt[`quarkus-smallrye-jwt`] provides support for authenticating using Json Web Tokens. It allows you to inject the token and claims into the application as per the MicroProfile JWT specification.
```

Comment:
I think the preferred way is to use Smallrye JWT (maybe in combination with Keycloak), as mentioned [here](https://github.com/devonfw-forge/devonfw-microservices/issues/17). So we should mention this first or make it more clear in some other way.